script/node_bs.py

```diff
@@ -173,11 +186,11 @@ def create_new_script():
         elif template_lines[i].startswith("PACKAGE_NAME"):
             template_lines[i]=f"PACKAGE_NAME={package_name}\n"
         
-    with open (f"{current_directory}/templates/build_script_node.sh",'w') as newfile:
+    with open (f"{current_directory}/templates/{script_language}",'w') as newfile:
```

Comment:
Can we simply use the destination path use in this script later (L192) here itself `f"{dir_name}/{package_name}_ubi_8.7.sh"`? It'll save the extra steps below. Only the command `new_cmd=f"python3 script/trigger_container.py -f templates/{script_language}"` (L193) we also need to be modified to use this path instead